spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/StandardizationExecution.scala

```diff
@@ -80,6 +81,11 @@ trait StandardizationExecution extends CommonJobExecution {
     // Add the raw format of the input file(s) to Atum's metadata
     Atum.setAdditionalInfo("raw_format" -> cmd.rawFormat)
 
+    val defaultTimeZoneForTimestamp = defaults.getDefaultTimestampTimeZone.getOrElse(spark.conf.get("spark.sql.session.timeZone"))
+    Atum.setAdditionalInfo("default_time_zone_for_timestamps"-> defaultTimeZoneForTimestamp)
+    val defaultTimeZoneForDate = defaults.getDefaultTimestampTimeZone.getOrElse(spark.conf.get("spark.sql.session.timeZone"))
+    Atum.setAdditionalInfo("default_time_zone_for_dates"-> defaultTimeZoneForDate)
```

Comment:
```suggestion
    val defaultTimeZoneForTimestamp = defaults.getDefaultTimestampTimeZone.getOrElse(spark.conf.get("spark.sql.session.timeZone"))
    Atum.setAdditionalInfo("default_time_zone_for_timestamps"-> defaultTimeZoneForTimestamp)
    val defaultTimeZoneForDate = defaults.getDefaultDateTimeZone.getOrElse(spark.conf.get("spark.sql.session.timeZone"))
    Atum.setAdditionalInfo("default_time_zone_for_dates"-> defaultTimeZoneForDate)
```

Haha! I have started testing this PR and without any config change, I got:
```
      "default_time_zone_for_dates": "UTC",
      "default_time_zone_for_timestamps": "UTC",
```	  
Then, I ran with conf override:
```--conf "spark.driver.extraJavaOptions=-DdefaultTimestampTimeZone=CET -DdefaultTimestampTimeZone=PST ..."```
and got:
```
      "default_time_zone_for_dates": "PST",
      "default_time_zone_for_timestamps": "PST",
```

That lead me to the 4 lines of StandardizationExecution that I originally considered fine. Hence the suggestion 😆

Reply:
- [x] code reviewed
- [x] pulled
- [x] built
- [x] run